### PR TITLE
Add rebeccapurple

### PR DESCRIPTION
**Pull request by clarcharr**
This was added in CSS4, and all of the recent versions of browsers support it.

### CODE REVIEW

**Comment by Ogeon** [comment]
Oh, nice! :smile: This looks great, but it seems like this library needs some tweaking, to make the tests pass again. Give me a couple of days and I'll see what state it is in, and how much work is required to bring it back to life.

**Comment by meh** [comment]
@Ogeon unrelated injection, but might give you some reason to keep working on this, I based my image handling [crate](https://github.com/meh/rust-picto) on this one for color handling because I found it really good.

So I'd really appreciate it if it was brought back to life 🐼

**Comment by Ogeon** [comment]
@meh That's great! I have been wanting to picking it up again, but the stars haven't really aligned. I have been caught up in various aspects of life for some time and my free time is still spotty, but I have recently been working a bit on my projects again. I'm sort of easing into it, so it may take some time.

My general plan is to mainly do maintenance, and not really add any new features to begin with. The goal is to make them easy to use and maintain, to avoid the situation I was in around the time this had to move to the back burner.

Long story short, I want to blow new life into Palette, but I don't want to make any detailed plans. I'll do it one step at the time, starting with making the tests pass.

**Comment by meh** [comment]
@Ogeon of course, no rush meant :)

If you need feedback or anything feel free to ping me!

**Comment by Ogeon** [comment]
>of course, no rush meant :)

I just wanted it to be clear from the start. :) It's simply an attempt to avoid mismatching expectations.

>If you need feedback or anything feel free to ping me!

Thanks, I'll also take a look at how your crate and see are using palette. That may give me some hints regarding what may need to be improved. :smile: Feel free to drop by with any feedback you happen to have, as well. It's always appreciated.

**Comment by clarcharr** [comment]
@Ogeon it appears that the failed build is just a linter warning that was promoted to an error by the tests. I added another commit to fix that.

In terms of the rest, one of the Srgb tests is failing due to a weird rounding error on my box, but the rest seem fine.

**Comment by Ogeon** [comment]
Hmm, yeah, looks like it's fine now. Rounding errors has plagued this library for a long time, and it's something that should be eventually taken care of, if they are still present. Let's see if Homu is still awake.

@homu r+

**Comment by Ogeon** [comment]
Ok, that works :) Thanks!
